wagit/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ project(wagit)
 
 set(CMAKE_CXX_STANDARD 20)
 
-add_executable(wagit main.cpp sources/wagit.cpp headers/wagit.h)
+add_executable(wagit main.cpp sources/WAGit.cpp headers/WAGit.h sources/Repository.cpp headers/Repository.h sources/GitObject.cpp headers/GitObject.h headers/GitConfig.h headers/GitExceptions.h sources/GitConfig.cpp includes/ini.h)
```

Comment:
Maybe creating some kind of sources files would be better here

Reply:
Yeah, that will happen. Wanted the project to get to a good state before I fix the wiring.